Commands.adoc

```diff
@@ -0,0 +1,239 @@
+manner that can be evaluated in a shell, use:
+-----------------------------------------------------------------------------
+$ vagrant service-manager env
+
+# docker env:
+# Set the following environment variables to enable access to the
+# docker daemon running inside of the vagrant virtual machine:
+export DOCKER_HOST=tcp://10.1.2.2:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# openshift env:
+# You can access the OpenShift console on: https://10.1.2.2:8443/console
+# To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+export OPENSHIFT_URL=https://10.1.2.2:8443
+export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+
+export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env)"
+-----------------------------------------------------------------------------
+
+If you need to run the plugin and get the environment variables and
+certificates for a specific provider, use: +
+For Docker:
+----------------------------------------------------------------------------
+$ vagrant service-manager env docker
+# Set the following environment variables to enable access to the
+# docker daemon running inside of the vagrant virtual machine:
+export DOCKER_HOST=tcp://172.28.128.3:2376
+export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+export DOCKER_TLS_VERIFY=1
+export DOCKER_API_VERSION=1.21
+
+# run following command to configure your shell:
+# eval "$(vagrant service-manager env docker)"
+----------------------------------------------------------------------------
+
+*Note:* In the case of the `docker` service, the required TLS certificates for
+securing the Docker communication will be regenerated and copied to the host
+as part of the command execution. If existing certificates are found to be
+invalid, they get regenerated.
+
+Similarly for OpenShift you can use: +
+`$ vagrant service-manager env openshift`
+
+If you want to run the plugin and get the environment variables and certificates
+ for docker in key=value format, use:
+----------------------------------------------------------------------
+$ vagrant service-manager env docker --script-readable
+DOCKER_HOST=tcp://172.28.128.3:2376
+DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker'
+DOCKER_TLS_VERIFY=1
+DOCKER_API_VERSION=1.21
+----------------------------------------------------------------------
+
+
+== Identify IP Address and Version of the VM
+
+If you need to find information related to the VM, such as the routable
+IP address or the version and release information, use:
+
+`$ vagrant service-manager box` +
+
+Usage: `vagrant service-manager box [sub-command] [options]`
+
+The following sub-commands can be used: +
+`ip`: Displays the routable IP address of the VM +
```

Comment:
as above. I'd try labeled list consistently and see how it goes. With only two options it is not as bad as the long options list in the beginning, but a consistent L&F would be good.